.github/workflows/puppet.yml

```diff
@@ -0,0 +1,47 @@
+name: Unit tests
+
+on: pull_request
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.4"
+    name: Puppet
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-${{ matrix.ruby }}-gems-unit-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-${{ matrix.puppet }}-${{ matrix.ruby }}-gems-unit-
+      - name: Install dependencies
+        run: |
+          sudo apt-get update && sudo apt-get install -y yajl-tools
+          gem install bundler
+          bundle config path vendor/bundle
+          cd puppet
+          bundle install --jobs 4 --retry 3
+      - name: Run tests
+        run: |
+          if [[ ! -f g10k ]] ; then
+            if [[ ! -f g10k-linux-amd64 ]] ; then
+              wget https://github.com/xorpaul/g10k/releases/download/v0.8.9/g10k-linux-amd64.zip
+            fi
+            unzip g10k-linux-amd64.zip
+          fi
+
+          ./g10k -quiet -cachedir .g10k/cache -puppetfile -puppetfilelocation puppet/Puppetfile_forge -moduledir puppet/forge_modules
+
+          cd puppet
+          bundle exec rake syntax
```

Comment:
```
---> syntax:hiera:yaml
ERROR: Failed to parse vendor/bundle/ruby/2.4.0/gems/puppet-6.19.1/spec/fixtures/unit/data_providers/environments/hiera_bad_syntax_yaml/data/bad.yaml: (vendor/bundle/ruby/2.4.0/gems/puppet-6.19.1/spec/fixtures/unit/data_providers/environments/hiera_bad_syntax_yaml/data/bad.yaml): found unexpected ':' while scanning a plain scalar at line 3 column 1
```

Seems it tries to parse the installed files and chokes on an explicitly bad one?

Reply:
Guess I need to find a way to ignore that path or to put that bundle path somewhere else